packages/react-examples/src/react-tooltip/Tooltip/Tooltip.stories.tsx

```diff
@@ -13,22 +13,17 @@ const useStyles = makeStyles({
 
   targetContainer: {
     display: 'inline-grid',
-    alignItems: 'center',
-    justifyItems: 'center',
+    gridTemplateColumns: 'repeat(5, 1fr)',
+    gridTemplateRows: 'repeat(5, 64px)',
     gap: '4px',
     margin: '16px 128px',
   },
-
-  target: {
-    width: '48px',
-    height: '48px',
-  },
 });
 
 export const Basic = () => {
   const styles = useStyles();
 
-  const targetRef = React.useRef<HTMLDivElement>(null);
+  const [exampleTarget, setExampleTarget] = React.useState<HTMLDivElement | null>(null);
```

Comment:
just an FYI, `useState` is not the only way to set a target, it's also possible to combine `useCallback` with `useRef` if you'd really like to avoid a rerender for this. 

Let's not advertise this so quickly now tho :)

Reply:
I think it wouldn't be possible to use a ref, since the `target` prop itself is not a ref. Changing the value of `target` will require a re-render anyways, and it's not possible to know the value of the ref on the initial render, right?

Previously, I had a prop `targetRef`, but I changed it to `target` be more consistent with Menu/Popover (and also because `targetRef` was a bit confusing, since ref props are usually "out" props, but I was using it as an "in" prop).

Reply:
Ah no I mean smth like this:

```tsx
const ref = React.useRef(null);
const callbackRef = React.useCallback(node => ref.current = node, [ref]);

<div ref={callbackRef} />
<Tooltip target={ref.current} /> // target should exist on initial render
```

just an FYI, it's really not use case we want to promote by default